ipywidgets/static/widgets/js/manager-base.js

```diff
@@ -386,13 +386,13 @@ define([
     };
 
     ManagerBase.prototype._create_comm = function(comm_target_name, model_id, metadata) {
-        throw new Error("Manager._create_comm not implemented");
+        return Promise.reject("No backend.");
     };
 
     ManagerBase.prototype._get_comm_info = function() {
-        throw new Error("Manager._get_comm_info not implemented");
+        return Promise.resolve({});
```

Comment:
I think this needs to be `return Promise.reject("No backend.");` because `_get_comm_info` is a method that returns a list of the existing comms from the backend.

If your problem with this is that you feel `set_state` should work without `_get_comm_info` being implemented, then `set_state` in the base should be modified to work without, and the one in the notebook WidgetManager should be implement so that it does.